src/styles/blocks/header.scss

```diff
@@ -39,7 +39,7 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
-    transition: 300ms;
+    transition: $smoothTransitionTime;
```

Comment:
Better to indicate `transition: color $smoothTransitionTime;` but its ok) Good job!)

Reply:
Yes, add property here.